src/PatientBrowserTutorial/index.html

```diff
@@ -0,0 +1,208 @@
+                    <th scope="col">Sex</th>
+                    <th scope="col">Birth Date</th>
+                    <th scope="col">Details</th>
+                    </tr>
+            </table>
+        </div>
+        <div class="rightContainer">
+            <pre id="json"></pre>
+        </div>
+    </div>
+
+<script>
+    var msalConfig = {
+        auth: {
+            clientId: config.clientId,
+            authority: config.authority
+        },
+        cache: {
+            cacheLocation: "localStorage",
+            storeAuthStateInCookie: true
+        }
+    }
+
+    var FHIRConfig = {
+        FHIRendpoint: config.fhirServerUrl
+    }
+
+    var requestObj = {
+        scopes: config.scopes
+    }
+    
+    var myMSALObj = new Msal.UserAgentApplication(msalConfig);
+    myMSALObj.handleRedirectCallback(authRedirectCallBack);
+    function authRedirectCallBack(error, response) {
+        if (error) {
+            console.log(error);
+        } else {
+            if (response.tokenType === "access_token") {
+                callFHIRServer(FHIRConfig.FHIRendpoint + '/Patient','GET', null, response.accessToken, FHIRCallback);
+            } else {
+                console.log("token type is:" + response.tokenType);
+            }
+        }
+    }
+    
+    function signIn() {
+        myMSALObj.loginPopup(requestObj).then(function (loginResponse) {
+            showWelcomeMessage();
+            document.getElementById('fileSelector')[0].style.visibility = "visible";
+            document.getElementById('myTable').style.visibility = "visible";
+            myMSALObj.acquireTokenSilent(requestObj).then(function (tokenResponse) {
+                callFHIRServer(FHIRConfig.FHIRendpoint + '/Patient','GET',null, tokenResponse.accessToken, FHIRCallback);
+            }).catch(function (error) {
+                console.log(error);
+                if (requiresInteraction(error.errorCode)) {
+                    myMSALObj.acquireTokenPopup(requestObj).then(function (tokenResponse) {
+                        callFHIRServer(FHIRConfig.FHIRendpoint + '/Patient','GET',null, tokenResponse.accessToken, FHIRCallback);
+                    }).catch(function(error) {
+                        console.log(error);
+                    })
+                }
+            });
+        }).catch(function(error) {
+            console.log(error);
+        })
+    }
+
+    function requiresInteraction(errorCode) {
+        if (!errorCode || !errorCode.length) {
+            return false;
+        }
+        return errorCode === "consent_required" ||
+            errorCode === "interaction_required" ||
+            errorCode === "login_required";
+    }
+
+    function signOut() {
+        myMSALObj.logout();
+    }
+
+    function FHIRCallback(json) {
+        for (var i = 0; i < Object.keys(json.entry).length; i++) {
+            json2table(json, i);
+            var display = JSON.stringify(json.entry[i].resource, null, 2);
```

Comment:
I am not sure it makes sense to embed the entire resource in each entry. Seems a bit hacky. Might be better to store the resources in a global variable and then just index from the button.